### DISCUSSION TITLE
Keep my saved queried (from original salesforce inspector)

### DISCUSSION THREAD

**Discussion by cmurad-tei** [tprouvot/Salesforce-Inspector-reloaded#523] [Q&A]
Is that possible for me to get the new inspector and keep the old one?
I have lots of saved queries and i do not want to lose them

**Comment by tprouvot** [comment]
Hi @cmurad-tei 
It is possible, I wrote a how to article in the documentation (documentation button in teb readme) . Let me know if you found it

**Comment by tprouvot** [comment]
@cmurad-tei  https://tprouvot.github.io/Salesforce-Inspector-reloaded/how-to/#migrate-saved-queries-from-legacy-extension-to-salesforce-inspector-reloaded
